go/logic/applier.go

```diff
@@ -1043,11 +1044,20 @@ func (this *Applier) ApplyDMLEventQueries(dmlEvents [](*binlog.BinlogDMLEvent))
 				if buildResult.err != nil {
 					return rollback(buildResult.err)
 				}
-				if _, err := tx.Exec(buildResult.query, buildResult.args...); err != nil {
+				result, err := tx.Exec(buildResult.query, buildResult.args...)
+				if err != nil {
 					err = fmt.Errorf("%s; query=%s; args=%+v", err.Error(), buildResult.query, buildResult.args)
 					return rollback(err)
 				}
-				totalDelta += buildResult.rowsDelta
+
+				rowsAffected, err := result.RowsAffected()
+				if err != nil {
+					log.Warningf("error getting rows affected from DML event query: %s. this might not be supported by go-sql-driver. i'm going to assume that the DML affected a row, but this may result in inaccurate statistics", err)
```

Comment:
@ajm188 if rows affected is not supported by the driver this will print a LOT of log messages during a migration

Would it be acceptable to silently default to `1` row affected here, ie: remove the `log.Warningf()`? cc @shlomi-noach for thoughts

Aside from resolving this question, this PR is ready to merge after a lot of successful tests 👍

Reply:
> if rows affected is not supported by the driver

I don't see that there should be a driver support. The driver we embed supports it. I think the error message is misleading. There could be an error because we're talking to a database and there can always be some connection issue. But the feature _is_ supported. So I can't see that this log will be spamming.

Reply:
Right, `database/sql` mentions that not all drivers support `.RowsAffected()` but because we hard-import the `go-sql-driver` that DOES support it this is a non-issue. Good point 👍 

In that case I suggest this change to the error:
```suggestion
					log.Warningf("error getting rows affected from DML event query: %s. i'm going to assume that the DML affected a single row, but this may result in inaccurate statistics", err)
```

cc @shlomi-noach and @ajm188

Reply:
works for me!